src/webportal/src/app/user/fabric/userView/utils.js

```diff
@@ -15,10 +15,6 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
-export const toBool = (val) => {
-  return (String(val)).toLowerCase() === 'true' || (String(val)).toLowerCase() === 'yes';
-};
-
 export const getVirtualCluster = (user) => {
-  return toBool(user.admin) ? 'All virtual clusters' : user.virtualCluster;
+  return user.admin ? 'All virtual clusters' : user.virtualCluster.slice().sort().join(', ');
```

Comment:
user.virtualCluster used here, but home page uses user.groupList.
I think we should unify the properties.